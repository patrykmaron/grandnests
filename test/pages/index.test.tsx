import React from 'react'
import { render } from '../testUtils'
import Home from '../../pages/index'
import { cleanup } from '@testing-library/react'

afterEach(cleanup)

describe('Home page', () => {
  it('matches snapshot and renders', () => {
    const { asFragment } = render(<Home />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
