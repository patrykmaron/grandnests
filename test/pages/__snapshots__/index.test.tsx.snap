// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot and renders 1`] = `
<DocumentFragment>
  <div
    class="styles__Wrapper-kt6arw-0 lcCCSV"
  >
    <div
      style="display: grid;"
    >
      header?
    </div>
    <div
      style="display: grid;"
    >
      main?
    </div>
    <div
      style="display: grid;"
    >
      footer?
    </div>
  </div>
</DocumentFragment>
`;
